index.html

```diff
@@ -0,0 +1,184 @@
+                    <img class="image cimage" src="" alt="">
+                    <div class="featur-box-content">
+                        <div class="feature-box-text cname"></div>
+                        <div class="feature-box-text1 cdescription"></div>
+                        <div class="feature-box-text2 coverview">
+                        </div>
+                    </div>
+
+                </div>
+                <div class="feature-box">
+                    <img class="image rimage" src="" alt="">
+                    <div class="feauture-box-content">
+                        <div class="feature-box-text rname"></div>
+                        <div class="feature-box-text1 rdescription"></div>
+                        <div class="feature-box-text2 roverview">.</div>
+                    </div>
+
+                </div>
+                <div class="feature-box">
+                    <img class="image simage" src="" alt="">
+                    <div class="feature-box-content">
+                        <div class="feature-box-text sname"></div>
+                        <div class="feature-box-text1 sdescription"></div>
+                        <div class="feature-box-text2 soverview">
+                        </div>
+                    </div>
+
+                </div>
+                <div class="feature-box last">
+                    <img class="image stimage" src="" alt="">
+                    <div class="feature-box-content">
+                        <div class="feature-box-text stname"></div>
+                        <div class="feature-box-text1 stdescription"></div>
+                        <div class="feature-box-text2 stoverview">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="partner">
+            <h1 class="partner-header">Sponsor</h1>
+            <ul class="partner-items">
+                <li class="tamar list">Tamar</li>
+                <li class="korg list">Korg</li>
+                <li class="zidijan list">Zidijan</li>
+                <li class="Nord list">Nord</li>
+                <li class="Yamaha list">Yamaha</li>
+            </ul>
+        </section>
+
+        <footer>
+            <div class="container">
+                 <div class="footer-logo">TGMC</div>
+                <p class="word"> The Global Musical Concert</p>
+            </div>
+
+            <div class="container2">
+                <p class="word2"> 2022 The Global Musical Concert. Some rights reserved</p>
+                <div class="text2">
+                    TGMC License                    
+                    All content on the website, unless otherwise specified, is available under a The Music Association Group (MAG) license. Please see
+                    for details about the concert information usage function details and permission conditions .
+                </div>
+            </div>
+        </footer>
```

Comment:
- [ ] The footer in the mobile version should be white as in the design please try to do that 
![image](https://user-images.githubusercontent.com/61790587/175137149-97b153fa-bb16-4f0d-ac79-d62f4e9f4960.png)